docs/sdk/src/guides/mod.rs

```diff
@@ -12,7 +12,7 @@
 //!
 //! > By this step, you have already launched a full Polkadot-SDK-based blockchain!
 //!
-//! Once done, feel free to step up into one of our templates: [`crate::polkadot_sdk::templates`].
+//! Once done, feel free to step up into one of our templates: [`templates`](crate::polkadot_sdk::templates).
```

Comment:
```suggestion
//! Once done, feel free to step up into one of our templates: [`templates`](crate::polkadot_sdk::templates).
```
```suggestion
//! Once done, feel free to step up into one of our templates: [`templates`].
```

So in all of them, you should now be able to remove the part that is inside the `()`.

Reply:
@kianenigma 
Thanks for your suggestion.
I have incorporated these changes. 
Let me know if thats fine or anything else needs to be changed further

Reply:
Hi @kianenigma  I am trying to get through pr checks for prdoc 
I am not sure what should go in crates because this script isn't impacting any crates but the documentation only. Initially I added substrate since it was giving me error later removed it than it asks for crates to be added , Kindly suggest what crates to be added in prdoc
`Required {
            path: "/crates",
        },`

Reply:
with the label I have added, it should be good.

Reply:
Thanks @kianenigma